### PR TITLE
Gujarati translations for stateSlug.js, about.js

### PR DESCRIPTION
**Pull request by technoknol**
Commits
----------
1. stateSlug.js for Gujarati. 
2. Fix translation for searchResFor.
3. Add Translation for a32, a112 and Fix a32 a33 swap issue.
4. Change double white spaces to single white spaces.
5. Gujarati Translation for data.js

### CODE REVIEW

**Review by kashyapshankhavara — APPROVED** [review]
searchResFor is for already displayed results.

**Comment by technoknol** [comment]
@kashyapshankhavara can you please review other commits ?

File: locales/gu/data.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-    govtContact: `Government Contact`,
-    govtDes: `Source, City, Contact`,
-    fillTheForm: `Fill the Form`
+    question: `શું તમારી પાસે ડેટા ચકાસેલ છે અને ડેટાબેસમાં ઉમેરવા માંગો છો?`,
+    sourceType: `સોર્સ, પ્રકાર, વિતરણ`,
+    medicineInjection: `દવા / ઇન્જેક્શન / કેમિસ્ટ`,
+    medicineDes: `વિતરક, વર્ગ, સરનામું`,
+    hospitalDes: `જીલ્લો, સરનામું, સંપર્ક`,
+    helplineDes: `વર્ગ, ફોન`,
+    doctorTele: `ડોક્ટર / ટેલિમેડિસિન / કન્સલ્ટેશન`,
```
Comment (nishitkamdaredb) [inline locales/gu/data.js:8]: About Consultation, can we use "સલાહ" or "પરામર્શ"

Comment (tecnoder) [inline locales/gu/data.js:8]: Can we use "પરામર્શ" or "સલાહ" instead?

File: locales/gu/data.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+    doctorDes: `વિશેષતા, શહેર, સંપર્ક`,
+    supportGroup: `સપોર્ટ જૂથો (વોટ્સઅપ, ટેલિગ્રામ વગેરે)`,
```
Comment (tecnoder) [inline locales/gu/data.js:10]: for this we can use "આધાર જૂથો"?

Comment (technoknol) [reply locales/gu/data.js]: Done

File: locales/gu/data.js (hunk 1 above)
Comment (technoknol) [reply locales/gu/data.js]: Done

**Comment by technoknol** [comment]
@Deep-Codes can you merge this PR now.

**Comment by Deep-Codes** [comment]
> @Deep-Codes can you merge this PR now.

Sure

**Pull request merged**